### ISSUE TITLE
Web Content Accessibility Guidelines; checkbox focus

### ISSUE THREAD

**Issue by quetzaluz** [react-toolbox/react-toolbox#1903]
I currently work for a health care company and we got a complaint in regards to WCAG (Web Content Accessibility Guidelines) for our checkboxes.

Our code currently applies the default checkbox focus styling seen here:
https://github.com/react-toolbox/react-toolbox/blob/73d5e082f1689579366af8bc7f4ac2a88c194124/components/checkbox/config.module.css#L8

However, the complain pertained to this being nearly invisible (invisible to the user who reported it) -- if possible the default should be darker for accessibility reasons.

Related Issues:
- Similar accessibility issue to https://github.com/react-toolbox/react-toolbox/issues/1887
- Bug in README.md: https://github.com/react-toolbox/react-toolbox/issues/1904

**Comment by quetzaluz** [comment]
Specific WCAG guideline: https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html

Which contains the specification that it should be highly visible: https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G195
